knowledge-base/security-vulnerability-wrd-r3-2020.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Security vulnerability present when using Web Report Designer
+description: A security vulnerability is introduced with version R3 2020
+type: how-to
+page_title: Security Vulnerability in R3 2020
+slug: security-vulnerability-wrd-r3-2020
+position: 
+tags: security,wrd,issue
+ticketid: 
+res_type: kb
+---
+
+## Environment
+<table>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Reporting</td>
+	</tr>
+	<tr>
+		<td>Project Type</td>
+		<td>Web Application</td>
+	</tr>
+	<tr>
+		<td>Report Dessgner</td>
+		<td>Web-based Report Designer</td>
+	</tr>
+</table>
+
+
+## Description
+A security vulnerability is introduced with version R3 2020 (14.2.20.916) and is present in web applications embedding the [Web-based Report Designer](https://docs.telerik.com/reporting/web-report-designer).
```

Comment:
I know that it is recommended to share as less information as possible in order to avoid exposing the vulnerability further, but can we at least share what data is vulnerable? This might motivate more people to upgrade.

Reply:
We discussed that and we prefer not to. Thanks for the input!